src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.css

```
@@ -24,17 +24,20 @@
 
 .optionsRow {
   display: grid;
-  grid-template-columns: repeat(2, 1fr);
+  grid-template-columns: repeat(3, auto);
+  column-gap: 32px;
   grid-template-rows: 1fr;
   align-items: center;
+  justify-content: end;
```

```suggestion
  justify-content: flex-end;
```

Not sure why the linter is not enforcing this one

---

What's the difference in grid layout?
In [CSS Box Alignment Module Level 3](https://www.w3.org/TR/css-align-3/#positional-values) it seems to say that unless in flex layout `flex-end` = `end`

For flex layout with `flex-direction: row-reverse` there is a difference between `justify-content: flex-end` and `justify-content: end` but in grid layout there is no property similar to `flex-direction` AFAIK
https://csslayout.news/whats-the-difference-between-the-alignment-values-of-start-flex-start-and-self-start/